src/components/templates/settingsLayout.tsx

```diff
@@ -28,12 +33,22 @@ export const SettingsLayout = () => {
 					<Sidebar />
 
 					<div className="flex flex-1 flex-col">
-						<TitleTopbar title={tSettings("title")} />
+						<TitleTopbar
+							title={
+								featureFlags.enableNewOrgsAndUsersDesign
+									? `${tSettings("user")}: ${user?.name || ""}`
+									: tSettings("title")
```

Comment:
In the new design, let's write just "Profile" instead of "User: [user name]"
```suggestion
									? `${tSettings("profile")}`
									: tSettings("title")
```

Reply:
changed